.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,41 @@
+---
+name: 'Bug Report'
+about: 'Report a bug, something does not work as it supposed to'
+title: '[bug] SHORT DESCRIPTION'
+labels: bug
+---
+
+<!--
+  Please don't forget to update the issue title.
+  Include all applicable information to help us reproduce your problem.
+
+  To help us debug your issue please explain:
+-->
+
+### Package and Environment Details (include every applicable attribute)
+  * Package Name/Version: **zlib/1.2.8**
+  * Operating System+version: **Linux Ubuntu 18.04**
+  * Compiler+version: **GCC 8**
+  * Docker image: **conanio/gcc8**
+  * Conan version: **conan 1.18.0**
+  * Python version: **Python 3.7.4**
+
+
+### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```

Comment:
Neither this.

Reply:
Use https://github.com/conan-io/conan/blob/develop/.github/CONTRIBUTING.md

Reply:
I've removed it from bug description.